### ISSUE TITLE
Old events and contacts inaccessible

### ISSUE THREAD

**Issue by LogicalDash** [etesync/android#166]
I can add new events to the calendar, and input new contacts, and they'll show up; but older contacts and events aren't showing up anymore. I do see them in the journal, if I view it in the EteSync app, but I can't see them in Calendar or Contacts.

This seems to be a regression, as old events and contacts do show up if I downgrade to EteSync 2.2.1.

I'm on Android 10 on a Moto g(7) Power.

**Comment by tasn** [comment]
Hey,

Thanks for reporting! Neither 2.2.2 nor 2.2.3 changed anything related. The closest thing that changed in 2.2.3 is how we handle newly created items, so I doubt it's related.

Could you walk me through what you did? Did you just upgrade -> it failed, and then downgraded -> it worked again? Or did you do anything else like removing and re-adding your account or something like that?

Thanks!

**Comment by LogicalDash** [comment]
I first noticed the error when I upgraded and my calendar widget was suddenly empty.

I tried removing and re-adding the account and the events appeared *for a moment* but when I looked again later they were gone. It seems inconsistent; I've removed and re-added three or so times and only got a glimpse of my events once.

I'm guessing the events are actually there somewhere and the EteSync provider isn't exposing them quite right?
